test/space/space_assertions.go

```diff
@@ -79,6 +79,13 @@ func (a *Assertion) HasTier(tierName string) *Assertion {
 	return a
 }
 
+func (a *Assertion) HasParentSpace(parentSpaceName string) *Assertion {
+	err := a.loadResource()
+	require.NoError(a.t, err)
+	assert.Equal(a.t, parentSpaceName, a.space.Spec.ParentSpace)
```

Comment:
how about checking also the label?

Reply:
added please see : [60159bc](https://github.com/codeready-toolchain/host-operator/pull/770/commits/60159bc49fc4872acb0b880853cb3775e5454300)